### PR TITLE
Pass over the back half

### PR DESCRIPTION
**Pull request by chris-wood**

### CODE REVIEW

File: draft-kuepau-iab-privacy-partitioning.md [hunk 1]
```diff
@@ -430,60 +437,121 @@ token issuance servers.
...
+| | Client |                 |        |   | Collector |    |
+| |        +---+             |        |   +-----+-----+    |
+| +--------+   |      +------V-----+  |         |          |
+|              +------>   Leader   <------------+          |
+|                     +------------+  |                    |
++-------------------------------------+--------------------+
+~~~
+{: #pa-topology title="Diagram of contexts in DAP"}
 
-# Analysis of Privacy Partioning
```
Comment (tfpauly) [inline draft-kuepau-iab-privacy-partitioning.md:443]: I'd still like to have a section for analysis, and not _just_ limits. I quite like the limits section, but I think there's still room for some general comments about "as new protocols come out, here's a guide to think about their use of partitioning".

So, could we add the stub back for now?

File: draft-kuepau-iab-privacy-partitioning.md [hunk 2]
```diff
@@ -430,60 +437,121 @@ token issuance servers.
...
+   set of unintentionally revealed information which cannot be guaranteed to be present and may disappear
+   any time in future.
+
+1. Varying performance effects. Depending on how context separation is done, privacy partitioning may
+   affect application performance. As an example, Privacy Pass introduces an entire end-to-end round
+   trip to issue a token before it can be redeemed, thereby decreasing perormance. In contrast, while
+   systems like CONNECT proxying may seem like they would regress performance, often times the highly
+   optimized nature of proxy-to-proxy paths leads to improved perforamnce. In general, while performance
+   and privacy tradeoffs are often cast as a zero sum game, in reality this is often not the case.
+   <!-- Do we want to same something about connection coalescing here as an example? -->
```
Comment (tfpauly) [inline draft-kuepau-iab-privacy-partitioning.md:540]: Are you thinking of how it is harder to coalesce over a proxy, or easier?

File: draft-kuepau-iab-privacy-partitioning.md (hunk 1 above)
Comment (chris-wood) [reply draft-kuepau-iab-privacy-partitioning.md:443]: I added back a stub with a slightly different title to better match what you have in mind. Have a look?

File: draft-kuepau-iab-privacy-partitioning.md (hunk 2 above)
Comment (tfpauly) [reply draft-kuepau-iab-privacy-partitioning.md:544]: Ah, I see. Maybe remove this in the text and file an issue for us to discuss?

Comment (chris-wood) [reply draft-kuepau-iab-privacy-partitioning.md]: Done!

**Pull request merged**